### PR TITLE
Setting job from stdin. Bug fix: dict -> CommentedMap

### PR DESCRIPTION
**Pull request by michael-kotliar**
When setting job from stdin `yaml.load` function is supposed to return `CommentedMap`, but returns `dict`. To return `CommentedMap` `RoundTripLoader` should be used.

### CODE REVIEW

**Comment by mr-c** [comment]
Jenkins, add to test list

File: cwltool/main.py [hunk 1]
```diff
@@ -414,8 +414,8 @@ def load_job_order(args, t, stdin, print_input_deps=False, relative_deps=False,
     if len(args.job_order) == 1 and args.job_order[0][0] != "-":
         job_order_file = args.job_order[0]
     elif len(args.job_order) == 1 and args.job_order[0] == "-":
-        job_order_object = yaml.load(stdin)
-        job_order_object, _ = loader.resolve_all(job_order_object, "")
+        job_order_object = yaml.load(stdin, yaml.RoundTripLoader)
```
Comment (tetron) [inline cwltool/main.py]: How about `yaml.round_trip_load()` instead

File: cwltool/main.py [hunk 2]
```diff
@@ -414,8 +414,8 @@ def load_job_order(args, t, stdin, print_input_deps=False, relative_deps=False,
...
+        job_order_object, _ = loader.resolve_all(job_order_object, 'file://%s' % os.path.abspath('.'))
```
Comment (tetron) [inline cwltool/main.py]: For the second parameter, use: `file_uri(os.getcwd()) + "/"`

File: cwltool/main.py (hunk 1 above)
Comment (michael-kotliar) [inline cwltool/main.py]: Done

File: cwltool/main.py (hunk 2 above)
Comment (michael-kotliar) [inline cwltool/main.py]: Done

File: cwltool/main.py (hunk 1 above)
Comment (michael-kotliar) [inline cwltool/main.py]: Should I change it back to `yaml.load(stdin, yaml.RoundTripLoader)`?
`yaml.round_trip_load()` causes an error in mypy check on Travis

Comment (mr-c) [inline cwltool/main.py]: @michael-kotliar Yeah, that's fine to switch it back for now.

**Comment by tetron** [comment]
Will merge this in #391

**Pull request closed without merging**